xml/System.Net/IPEndPoint.xml

```diff
@@ -602,10 +604,30 @@
         <Parameter Name="s" Type="System.String" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="s">A <see cref="T:System.String" /> that contains an IP endpoint in dotted-quad notation or network byte order for IPv4 and in colon-hexadecimal notation for IPv6.</param>
+        <summary>Converts an IP network endpoint (address and port) represented as a <see cref="T:System.String" /> to an IPEndPoint instance.</summary>
+        <returns>An <see cref="T:System.Net.IPEndPoint" /> instance.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+The number of parts (each part is separated by a period) in the string s determines how the endpoint's IP address is constructed. A one part address is stored directly in the network address. A two part address, convenient for specifying a class A address, puts the leading part in the first byte and the trailing part in the right-most three bytes of the network address. A three part address, convenient for specifying a class B address, puts the first part in the first byte, the second part in the second byte, and the final part in the right-most two bytes of the network address. For Example:
+
+|Number of parts and example `s`|IPv4 address for IPEndPoint.Address|Port
+|-------------------------|--------------------------------|-------------------------|  
+|1 -- "1"|0.0.0.1|0|  
+|2 -- "20.2:80"|20.0.0.2|80|  
+|2 -- "20.65535:23"|20.0.255.255|23|  
+|3 -- "128.1.2:443"|128.1.0.2|443|  
+
+Note that this method accepts as valid an string value that can be parsed as an Int64, and then treats that Int64 as the long value of an IP address in network byte order, similar to the way that the IPAddress constructor does. This means that this method returns true if the Int64 is parsed successfully, even if it represents an address that's not a valid IP address. For example, if s is "1", this method returns true even though "1" (or 0.0.0.1) is not a valid IP address and you might expect this method to return false. Fixing this bug would break existing apps, so the current behavior will not be changed. Your code can avoid this behavior by ensuring that it only uses this method to parse IP addresses in dotted-decimal format.
```

Comment:
This seems like an important note.

```suggestion
> [!IMPORTANT]
> Note that this method accepts as valid an string value that can be parsed as an <xref:System.Int64>, and then treats that <xref:System.Int64> as the long value of an IP address in network byte order, similar to the way that the <xref:System.Net.IPAddress> constructor does. This means that this method returns `true` when the <xref:System.Int64> is parsed successfully, even if it represents an address that's not a valid IP address. For example, if `s` is "1", this method returns `true` even though "1" (or 0.0.0.1) is not a valid IP address. Fixing this bug would break existing apps, so the current behavior will not be changed. Your code can avoid this behavior by ensuring that it only uses this method to parse IP addresses in dotted-decimal format.
```